themes/10up-theme/package.json

```diff
@@ -2,26 +2,26 @@
   "name": "tenup-theme",
   "version": "1.0.0",
   "scripts": {
-    "start": "10up-scripts start",
-    "watch": "10up-scripts watch",
-    "build": "10up-scripts build",
-    "format-js": "10up-scripts format-js",
-    "lint-js": "10up-scripts lint-js",
-    "lint-style": "10up-scripts lint-style",
-    "test": "10up-scripts test-unit-jest",
+    "start": "10up-toolkit start",
+    "watch": "10up-toolkit watch",
```

Comment:
The build command also supports a `--watch` flag. So it's also possible to do `"watch": "npm run build -- --watch"`

Reply:
@nicholasio what is the difference between `watch` and the existing `start` command?

Reply:
none, they're the same. `10up-toolkit start` `'10up-toolkit build --watch` and `10up-toolkit watch` is the same command.